docs/en/observability/logs-filter.asciidoc

```diff
@@ -0,0 +1,334 @@
+[[logs-filter-and-aggregate]]
+= Filter and aggregate logs
+
+Filter and aggregate your log data to find specific information, gain insight, and monitor your systems more efficiently. You can filter and aggregate based on structured fields like timestamps, log levels, and IP addresses that you've extracted from your log data. If still need to parse your log data and extract structured fields, see the <<logs-stream-parse>> documentation.
+
+See the following sections for more on filtering and aggregating your log data:
+
+* <<logs-filter>> — Narrow down your log data by applying specific criteria.
+* <<logs-aggregate>> — Analyze and summarize data to find patterns and gain insight.
+
+[discrete]
+[[logs-filter-and-aggregate-prereq]]
+== Before you get started
+
+The examples on this page use the following ingest pipeline and index template, which you can set in *Dev Tools*. For more on using ingest pipelines and index templates with your log data, see the <<logs-stream-parse>> documentation.
+
+Set the ingest pipeline with the following command:
+
+[source,console]
+----
+PUT _ingest/pipeline/logs-example-default
+{
+  "description": "Extracts the timestamp log level and host ip",
+  "processors": [
+    {
+      "dissect": {
+        "field": "message",
+        "pattern": "%{@timestamp} %{log.level} %{host.ip} %{message}"
+      }
+    }
+  ]
+}
+----
+
+Set the index template with the following command:
+
+[source,console]
+----
+PUT _index_template/logs-example-default-template
+{
+  "index_patterns": [ "logs-example-*" ],
+  "data_stream": { },
+  "priority": 500,
+  "template": {
+    "settings": {
+      "index.default_pipeline":"logs-example-default"<4>
+    }
+  },
+  "composed_of": [
+    "logs-mappings",
+    "logs-settings",
+    "logs@custom",
+    "ecs@dynamic_templates"
+  ],
+  "ignore_missing_component_templates": ["logs@custom"],
+}
+----
+
+[discrete]
+[[logs-filter]]
+== Filter logs
+
+Filter your data using the fields you've extracted so you can focus on log data with specific log levels, timestamp ranges, or host IPs. You can filter your log data in different ways:
+
+- <<logs-filter-logs-explorer>> – Filter and visualize log data in {kib} using Logs Explorer.
+- <<logs-filter-qdsl>> – Filter log data from Dev Tools using Query DSL.
+
+[discrete]
+[[logs-filter-logs-explorer]]
+=== Filter logs in Logs Explorer
+
+Logs Explorer is a {kib} tool that automatically provides views of your log data based on integrations and data streams. You can find Logs Explorer in the Observability menu under *Logs*. Use the {kibana-ref}/kuery-query.html[{kib} Query Language (KQL)] in the search bar to narrow down the log data displayed in Logs Explorer.
+
+For example, you might want to look into an event that occurred within a specific time range (between September 14th and 15th).
+
+First, from *Dev Tools* add some logs with varying timestamps and log levels to your data stream with the following command:
```

Comment:
It might be good to reorient the user since we're on a new page:

```suggestion
Add some logs with varying timestamps and log levels to your data stream:

. In {kib}, go to *Management* -> *Dev Tools*.
. In the *Console* tab, run the following command:
```